app/main/routes.py

```diff
@@ -27,16 +29,44 @@ def upload():
             return render_template("upload.html", response=response)
 
         # TODO: Update this to round_four when available
-        response = post_ingest(excel_file, {"source_type": "tf_round_three"})
-        response = response.json()
+        # response = post_ingest(excel_file, {"source_type": "tf_round_three"})
+        # response_a = response.json()
+
+        # TODO: REMOVE THIS ONCE BACK END IS RE-ATTACHED !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+        with open('tests/resources/validation_errors.json') as error_json:
+            response = json.load(error_json)
+        # TODO !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```

Comment:
Unit tests run independently of other repositories so these should remain valid.

Reply:
Ah, they're in routes.py. 

Remove these from routes.py and mock them in a test either with the dummy files or by putting the dummy info into a pyfixture somewhere.